### PR TITLE
restaurant create form, must be a user to create restaurant

### PR DESCRIPTION
**Pull request by ddavebbell**

### CODE REVIEW

File: app/controllers/reservations_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class ReservationsController < ApplicationController
+	include TimeUtility
```
Comment (somethinrother) [inline app/controllers/reservations_controller.rb]: remove this

File: config/application.rb [hunk 2]
```diff
@@ -10,7 +10,6 @@ module Seatyourself
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.0
```
Comment (somethinrother) [inline config/application.rb]: Add a new line back here

File: app/controllers/reservations_controller.rb (hunk 1 above)
Comment (ddavebbell) [reply app/controllers/reservations_controller.rb]: i just did

File: config/application.rb (hunk 2 above)
Comment (ddavebbell) [reply config/application.rb]: done

Comment (ddavebbell) [reply config/application.rb]: where does the line go? I pushed new changes

**Pull request merged**